test/pet.test.js

```diff
@@ -1,24 +1,246 @@
 const Pet = require('../src/pet');
 
-describe('constructor', () => {
-  let pet;
+
+  let petCat;
 
   beforeEach(() => {
-     pet = new Pet('Rocky');
-     expect(new Pet('Rocky')).toBeInstanceOf(Object);
-     expect(pet.name).toEqual('Rocky');
+     petCat = new Pet('Rocky');    
 
   });
 
-  it('has a initial age of 0 & hunger of 0 & fitness of 10', () => {
+describe('constructor', () => {
 
-      expect(pet.age).toEqual(0);
-      expect(pet.hunger).toEqual(0);
-      expect(pet.fitness).toEqual(10);
-       
-    });
+  it ('Creates petCat as an instance of an Object', () => {
+    expect(new Pet('Rocky')).toBeInstanceOf(Object);
+
+  });
+
+  it ('petCat has the specified name', () => {
+
+    petCat.name = 'Rocky';
```

Comment:
This test is not testing that the Pet class worked because you are assigning the petCat.name to Rocky and then testing petCat.name === 'Rocky'. so in this case it should be enough not to make petCat.name = 'Rocky' on line 20